operations/service_runner.go

```diff
@@ -29,8 +29,6 @@ import (
 	"github.com/urfave/cli"
 )
 
-const useNewScheduler = true
```

Comment:
Thereis still 1 place that uses this variable

Reply:
fixed.